src/components/DistanceRequest/index.js

```diff
@@ -130,6 +145,22 @@ function DistanceRequest({transactionID, report, transaction, route, isEditingRe
         Navigation.navigate(isEditingRequest ? ROUTES.MONEY_REQUEST_EDIT_WAYPOINT.getRoute(report.reportID, transactionID, index) : ROUTES.MONEY_REQUEST_WAYPOINT.getRoute('request', index));
     };
 
+    const getError = () => {
```

Comment:
Duplicate waypoint is not showing error when rearranged, ref: https://github.com/Expensify/App/pull/35910